source/MapPanel.cpp

```diff
@@ -578,6 +585,10 @@ bool MapPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool
 			this, &MapPanel::Find, "Search for:", "", Truncate::NONE, true));
 		return true;
 	}
+	else if(key == ']' && !mapIsStarry)
+		player.SetStarryMap(true);
+	else if(key == '[' && mapIsStarry)
+		player.SetStarryMap(false);
```

Comment:
Is there a reason why this can't be a toggle using a single key? Also, this isn't documented anywhere in the wiki pr, I believe. (Which might be useful, as this doesn't get the relevant character underlined in the UI.)